homeassistant/components/aurora_abb_powerone/strings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Configure Aurora ABB PowerOne solar PV inverter",
+        "description": "Select configuration options. Inverter must be connected via an RS485 adaptor, please select serial port and the inverter's address as configured on the LCD panel",
```

Comment:
```suggestion
        "description": "The inverter must be connected via an RS485 adapter. Select the serial port and the inverter's address as configured on the LCD panel.",
```

Reply:
I agree with the edits, but any time I change anything in strings.json, when I try to commit i get this (what does it mean?):
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/workspaces/core/script/hassfest/__main__.py", line 7, in <module>
    from . import (
  File "/workspaces/core/script/hassfest/dependencies.py", line 6, in <module>
    from homeassistant.requirements import DISCOVERY_INTEGRATIONS
  File "/workspaces/core/homeassistant/requirements.py", line 6, in <module>
    from homeassistant.core import HomeAssistant
  File "/workspaces/core/homeassistant/core.py", line 41, in <module>
    from homeassistant import block_async_io, loader, util
  File "/workspaces/core/homeassistant/loader.py", line 31, in <module>
    from awesomeversion import AwesomeVersion, AwesomeVersionStrategy
ModuleNotFoundError: No module named 'awesomeversion'
```

Reply:
Might be your virtualenv and/or devcontainer isn't up to date - awesomeversion was introduced when the versioning of Core switched from semantic to awesome (e.g. 0.99.3 to 2021.2.1)? Just doublecheck the developers-site for how to setup the environment, and/or just install it using `pip install awesomeversion==21.2.3`

But you might run into more issues if your dependencies aren't up to date

Reply:
> Might be your virtualenv and/or devcontainer isn't up to date - awesomeversion was introduced when the versioning of Core switched from semantic to awesome (e.g. 0.99.3 to 2021.2.1)? Just doublecheck the developers-site for how to setup the environment, and/or just install it using `pip install awesomeversion==21.2.3`
> 
> But you might run into more issues if your dependencies aren't up to date

@CoMPaTech Thanks for this.  I seem to waste a huge amount of time just getting the environment to work.  So far I've tried 
```
pip install awesomeversion==21.2.3
pip install -r requirements_test_all.txt -c homeassistant/package_constraints.txt
git fetch upstream dev
git rebase upstream/dev
(rebuild dev container)
pip install -r requirements_test_all.txt -c homeassistant/package_constraints.txt
script/setup
```
but still git commit doesn't work, exact same error.

I notice that
`script/run-in-env.sh python3 -m script.hassfest` 
fails, but
`python3 -m script.hassfest`
works

Is there a single script that will rip up my whole devcontainer and download everything again, then set up all dependencies for development and test?  I feel I need this!

Reply:
If you're in vsc/devcontainer you should be able to do ctrl-shift-p or something (don't recall the right one, might be shift-p) and search for 'rebuild container' - or throw away the container so it will rebuild. I threw away container on my laptop and it rebuilds everything nicely automatically.

Reply:
Yes, I was already doing that "(rebuild dev container)" in above, no dice. I think I've solved it though.

After also doing 
```
docker ps -a
docker ps rm <image name>
rm -rf venv
```
Then it worked!

Reply:
👍

Reply:
Now fixed.